src/state/url-sync-mixin.ts

```diff
@@ -25,6 +25,9 @@ export const urlSyncMixin = <
 
         public connectedCallback(): void {
           super.connectedCallback();
+          if (history.length === 1) {
```

Comment:
What about if a user enters on a sub-page via a My link? Shouldn't we just always specify "root", regardless of history length ?

Reply:
Bram told me this breaks refreshing, so no.